### ISSUE TITLE
Weird escape sequence at the beginning of enju's output

### ISSUE THREAD

**Issue by rbossy** [mynlp/enju#11]
I have noticed that sometimes enju's output stream starts with the sequence `\033[?1034h` (and breaks scripts that parse it).

A quick search reveals that this is a terminal escape sequence for handling the meta key.

I'm almost sure that this is not a bug for enju but rather a bug of the I/O system libraries it uses. I'm leaving it here for documentation purposes.

**Comment by miyao-yusuke** [comment]
Exactly.  It appears this happens when the environment variable `TERM` is `xterm` for some version of OS.  A compromise solution is to specify another value to `TERM`, e.g. `TERM=vt100`.
